keras_cv/training/contrastive/contrastive_trainer.py

```diff
@@ -110,9 +110,15 @@ def __init__(
             self.probing_top = layers.Dense(classes, name="linear_probe")
 
     def compile(
-        self, probe_optimizer=None, probe_loss=None, probe_metrics=None, **kwargs
+        self,
+        optimizer,
+        loss,
```

Comment:
Any reason to take optimizer and loss here at all?  From my understanding  we may as well keep them in kwargs

Reply:
No, there isn't a reason. This was a mistake -- done.